src/pages/test-results-search/test-results-search.scss

```diff
@@ -66,4 +66,36 @@ page-test-results-search {
     margin-right: 0;
     width: 100%;
   }
+
+  .search-inputs {
+    position: absolute;
+    width: 800px;
+    z-index: 100;
+  }
+
+  @media screen and (min-width: 900px) {
+    .search-inputs {
+      position: absolute;
+      width: 1080px;
+      z-index: 100;
```

Comment:
Did we need still this z-index then?

Reply:
ignore me, thought this was .list-header

Reply:
The z-index would bring the search tabs in from of the the ion-list header.